indra/sources/omnipath/processor.py

```diff
@@ -0,0 +1,175 @@
+from __future__ import unicode_literals
+import logging
+from collections import Counter
+from indra.preassembler.grounding_mapper import GroundingMapper
+from indra.statements import modtype_to_modclass, Agent, Evidence, Complex, \
+    get_statement_by_name as stmt_by_name, BoundCondition
+
+logger = logging.getLogger(__file__)
+
+
+class OmniPathProcessor(object):
+    def __init__(self, ptm_json=None, ligrec_json=None):
+        self.statements = []
+        self.statements.extend(self._stmts_from_op_mods(ptm_json))
+        self.statements.extend(self._stmt_from_op_lr(ligrec_json))
```

Comment:
In many other processors, these two calls are actually done in the API so that the processor can be constructed without immediately starting processing. However, we can leave it like this too, not a big issue.